core/templates/dev/head/filtersSpec.js

```diff
@@ -410,4 +410,79 @@ describe('Testing filters', function() {
     expect(filter('  only First lettEr is  Affected ')).toEqual(
       'Only First lettEr is  Affected');
   }));
+
+  it('should correctly display RTE components', inject(function($filter) {
```

Comment:
Add tests for when you have 2 images in components and if there are mixed types (ie. image and video).